plugin/bookmark.vim

```diff
@@ -491,6 +486,26 @@ function! s:is_quickfix_win()
   return getbufvar(winbufnr('.'), '&buftype') == 'quickfix'
 endfunction
 
+function! s:show_location()
+  if g:bm_stack_mode
```

Comment:
I like that you extracted this into it's own function as it gets more complicated now. If we could just parametrize `bm#location_list()` this whole function would become more readable.

Untested code:
```
function! s:show_location()
  lgetexpr bm#location_list(g:bm_stack_mode)
  if g:bookmark_location_list
    belowright lopen
  else
    belowright copen
  endif
endfunction
```